### DISCUSSION TITLE
adblock logger

### DISCUSSION THREAD

**Discussion by dashingdon** [qutebrowser/qutebrowser#8272] [Q&A]
anyways to see what is getting blocked and by which rule (host list or ABP rule) when we use adblocker?  thank you!

**Comment by The-Compiler** [comment]
Running qutebrowser with `--debug --logfilter network` (or looking at those entries in `:messages debug`) will show what's being blocked, and so will the network tab in `:devtools`. No way to see which rule caused that to happen, however.
